tools/dds/dds-server/dds-participant.h

```diff
@@ -0,0 +1,31 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+namespace eprosima {
+namespace fastdds {
+namespace dds {
+class DomainParticipant;
+}
+}  // namespace fastdds
+}  // namespace eprosima
+
+class dds_participant_listener;
+
+namespace tools {
```

Comment:
tools?

Reply:
Hmm I see this is the namespace for the existing `dds_server` class...
Is this because we thought it would be for the `rs-dds-server` tool? What do you think, shouldn't this be inside a `librealsense::dds` namespace?

Reply:
You mean move it to `librealsense/include` ?
Cause otherwise we cannot access it from the tool.